explore/src/main/scala/explore/config/sequence/GeneratedSequenceTables.scala

```diff
@@ -22,20 +21,18 @@ object GeneratedSequenceTables:
   private val component =
     ScalaFnComponent
       .withHooks[Props]
-      .render { props =>
-        <.div(^.height := "100%", ^.overflow.auto)(
```

Comment:
Nice there are done now in css?

Reply:
Turns out it's not necessary anymore. I assume it was in the migration to Prime.